### PR TITLE
PB-3854: Patch job name and nfs export path 

### PR DESCRIPTION
**Pull request by pallav-px**
**What this PR does / why we need it**:
fixes the issue related to nfscsirestore taking wrong export and sub path. Also patched the name for nfscsirestore job
**Which issue(s) this PR fixes** (optional)
Closes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by pallav-px** [comment]
Screenshot 2023-05-12 at 1 23 44 PM 1
  Now the job name starts with nfscsirestore before it was restore-backupUID-VolumeUID

File: Makefile [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+RELEASE_VER ?= nfs-dev
 BUILD_DATE  := $(shell date -u +'%Y-%m-%dT%H:%M:%SZ')
 BASE_DIR    := $(shell git rev-parse --show-toplevel)
 GIT_SHA     := $(shell git rev-parse --short HEAD)
 BIN         := $(BASE_DIR)/bin
 
 DOCK_BUILD_CNT  := golang:1.19.1
 
-DOCKER_IMAGE_REPO?=portworx
+DOCKER_IMAGE_REPO?=pallavpx
```
Comment (siva-portworx) [inline Makefile:9]: revert this change.

File: Makefile [hunk 2]
```diff
@@ -1,12 +1,12 @@
-RELEASE_VER ?= 1.2.6-dev
+RELEASE_VER ?= nfs-dev
```
Comment (siva-portworx) [inline Makefile:1]: revert this change

**Review by siva-portworx — APPROVED** [review]

File: Makefile (hunk 2 above)
Comment (pallav-px) [reply Makefile]: done
